### PR TITLE
Consistify help output

### PR DESCRIPTION
**Pull request by mikethebeer**
```
$ croud -h       
usage: croud [-h] [--version] {login,logout,me} ...

A command line interface for CrateDB Cloud ☁

positional arguments:
  {login,logout,me}
    login            Performs an OAuth2 Login to CrateDB Cloud
    logout           Performs a logout of the current logged in User
    me               Prints the current logged in user

optional arguments:
  -h, --help         show this help message and exit
  --version          show program's version number and exit
```

```
$ croud --version
croud 0.2.1.dev3+gdf82e23.d20181130
```

### CODE REVIEW

File: croud/__main__.py [hunk 1]
```diff
@@ -33,7 +33,9 @@ def main():
     Configuration.create()
     colorama.init()
 
-    p = argh.ArghParser(prog="CROUD")
+    p = argh.ArghParser(
+        prog="croud", description="A command line interface for CrateDB Cloud ☁"
```
Comment (jodok) [inline croud/__main__.py]: out of curiosity: what is that symbol at the end?

Comment (MarkusH) [reply croud/__main__.py]: A :cloud:

Comment (jodok) [reply croud/__main__.py]: looks more like a birdshit :)

Comment (mikethebeer) [reply croud/__main__.py]: tbh I'm fine to remove it 💩

**Pull request merged**